src/Plugin/views/field/OgMembershipBulkForm.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\og\Plugin\views\field;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\og\OgMembershipInterface;
+use Drupal\system\Plugin\views\field\BulkForm;
+
+/**
+ * Defines a user operations bulk form element.
+ *
+ * @ViewsField("og_membership_bulk_form")
+ */
+class OgMembershipBulkForm extends BulkForm {
+
+  /**
+   * {@inheritdoc}
+   *
+   * Provide a more useful title to improve the accessibility.
+   */
+  public function viewsForm(&$form, FormStateInterface $form_state) {
+    // Remove all actions related to (non-)member role.
+    foreach ($this->actions as $key => $action) {
+      if (strstr($key, '-member') !== FALSE) {
```

Comment:
So the actions to change the membership role to `member` or `non-member` are created first, and then filtered in the bulk form. It makes more sense to not make these actions in the first place. They are not useful and then we don't need to filter them either.

Instead of having an action to change the membership of a user to `non-member` it would be better to have a dedicated action `og_membership_remove`.